web/web/src/app/metalakes/metalake/rightContent/tabsContent/tableView/TableView.js

```diff
@@ -251,31 +262,35 @@ const TableView = () => {
             title='Details'
             size='small'
             sx={{ color: theme => theme.palette.text.secondary }}
-            onClick={() => handleShowDetails({ row, type: 'catalog' })}
-            data-refer={`view-catalog-${row.name}`}
+            onClick={() => handleShowDetails({ row, type: row.node })}
+            data-refer={`view-entity-${row.name}`}
           >
             <ViewIcon viewBox='0 0 24 22' />
           </IconButton>
 
-          <IconButton
-            title='Edit'
-            size='small'
-            sx={{ color: theme => theme.palette.text.secondary }}
-            onClick={() => handleShowEditDialog({ row, type: 'catalog' })}
-            data-refer={`edit-catalog-${row.name}`}
-          >
-            <EditIcon />
-          </IconButton>
-
-          <IconButton
-            title='Delete'
-            size='small'
-            sx={{ color: theme => theme.palette.error.light }}
-            onClick={() => handleDelete({ name: row.name, type: 'catalog', catalogType: row.type })}
-            data-refer={`delete-catalog-${row.name}`}
-          >
-            <DeleteIcon />
-          </IconButton>
+          {!isKafkaSchema && (
+            <IconButton
+              title='Edit'
+              size='small'
+              sx={{ color: theme => theme.palette.text.secondary }}
+              onClick={() => handleShowEditDialog({ row, type: row.node })}
+              data-refer={`edit-entity-${row.name}`}
+            >
+              <EditIcon />
+            </IconButton>
+          )}
+
+          {!isKafkaSchema && (
+            <IconButton
+              title='Delete'
+              size='small'
+              sx={{ color: theme => theme.palette.error.light }}
+              onClick={() => handleDelete({ name: row.name, type: row.node, catalogType: row.type })}
+              data-refer={`delete-entity-${row.name}`}
+            >
+              <DeleteIcon />
+            </IconButton>
```

Comment:
Currently, we cannot edit or delete hudi schema, am I right @mchades ?

Reply:
yes

Reply:
The version of this PR base does not yet support the hudi catalog, I can submit another pr for that later.